core/src/test/java/io/aiven/klaw/UtilMethods.java

```diff
@@ -180,6 +171,33 @@ public List<UserInfoModelResponse> getUserInfoListModel(String username, String
     return userInfoList;
   }
 
+  public Team getTeamDaoMock() {
+    ServiceAccounts serviceAccounts = new ServiceAccounts();
+    serviceAccounts.setNumberOfAllowedAccounts(-1);
+
+    Team team = new Team();
+    team.setTeamname("Seahorses");
+    team.setTeamId(101);
+    team.setContactperson("Contact Person");
+    team.setTenantId(101);
```

Comment:
Can you make the tenant id a class static variable, as 101, I know that there are a lot of other instances of the tenant id being set here but we should be using a consistent way of doing this.